app/helpers/questions_helper.rb

```diff
@@ -6,7 +6,20 @@ def get_question_list
     JSON.parse(RestClient.get(ENV["qm_api_url"] + "questions"))
   end
 
+  def destroy_question question
+    json = question.to_json(:include => :answers)
```

Comment:
Do you need to store the question in JSON format? If not, you could delete this and the debug line below. (Doesn't look like it's used anywhere -- all you need for the `RestClient.delete()` call is the ID, which you're getting directly from the params anyway.